articles/connections/database/mysql.md

```diff
@@ -79,6 +79,10 @@ This script connects to a **MySQL** database and executes a query to retrieve th
 
 If you are using [IBM's DB2](https://www.ibm.com/analytics/us/en/technology/db2/) product, [click here](/connections/database/db2-script) for a sample login script.
 
+::: panel-info
+You can use the `auth0-custom-db-testharness` [library](https://www.npmjs.com/package/auth0-custom-db-testharness) to deploy, execute, and test the output of Custom DB Scripts using a Webtask sandbox environment.
+:::
+
```

Comment:
@khorne3 I don't know why this keeps looking like this:

![image](https://cloud.githubusercontent.com/assets/11715799/23457577/299e70be-fe81-11e6-9a1a-0925090ff0f7.png)

Maybe it's the missing title? When I'm adding it usually it gets fixed. Anyway, I will replace this with an [alert](https://github.com/auth0/docs/blob/master/CONTRIBUTING.md#alerts). I think it looks better for short notes, wdyt?